scipy/interpolate/tests/test_interpolate.py

```diff
@@ -2546,7 +2546,7 @@ def __init__(self, shape):
         self._v = np.arange(np.prod(shape)).reshape(shape)
 
     def __getitem__(self, idx):
-        return self._v[idx]
+        return self._v[tuple(idx)]
```

Comment:
Might make more sense to let the error propagate here, and fix it at the caller instead

Reply:
Actually, I failed to trigger this warning with the numpy HEAD, so I've just reverted this change.